adi/assessment_data_import.py

```diff
@@ -147,6 +147,32 @@ def import_data(
 
         # Iterate through assessment data and save each record to the database
         for assessment in assessment_data:
+            # Check for the most required of fields
+            if "id" not in assessment:
+                logging.warning("Assessment missing 'id'! Skipping...")
```

Comment:
When the `id` is missing, we should output the assessment info that is present so that a human can figure out what went wrong.

Reply:
I only omitted record information because I wasn't sure if we wanted assessment information showing up in the CloudWatch log stream. If we didn't want that information in the log stream I could include the record indexes and output that information. That way a human could retrieve the JSON from S3 and look at the specified record index. Otherwise I can just output the record if it's missing an `id` key.

Reply:
Agreed- the record index should be sufficient.  Thanks!

Reply:
Please see e3a4b3e for adding the record index to the warning message.